### ISSUE TITLE
Deprecation warning during `udata init`

### ISSUE THREAD

**Issue by abulte** [opendatateam/udata#984]
## Detailed Description

During the creation of the admin user via `udata init`, a deprecation warning is issued:

```
Do you want to create a superadmin user ? [y] y
First name: Alexandre Admin
Last name: Admin
Email: alexandre@xxx.com
Password:
Confirm Password:
/Users/alexandre/Developer/Etalab/udata/pyenv/lib/python2.7/site-packages/flask_security/forms.py:267: FlaskWTFDeprecationWarning: "csrf_enabled" is deprecated and will be removed in 1.0. Set "meta.csrf" instead.
  super(RegisterForm, self).__init__(*args, **kwargs)

User created successfully
```

## Context

I do not want any warning, especially during my first interaction with the product;

## Possible Implementation

Follow the suggestions of the error.

## Your Environment

`dev` branch.

**Comment by abulte** [comment]
This should be fixed by https://github.com/mattupstate/flask-security/pull/660

**Comment by abulte** [comment]
And thus will be fixed by #958.
